### ISSUE TITLE
Longer load time

### ISSUE THREAD

**Issue by RobMatteson** [Truinto/DarkCodex#33]
Game stops loading around 60% when playing 1.2.0 version of game with 1.2.0 version of DarkCodex. Game loads ok when DarkCodex is uninstalled.

**Comment by Truinto** [comment]
I have a couple features that read through all blueprints. Way back that wasn't an issue as all blueprints are loaded into the game anyway. This was optimized some time ago, which shortens load time noticeable. Unfortunately my mod now takes more time to load (exactly the time which was gained by the optimization). On the plus side, the blueprints stay in cache, so this time loss is only once, even if other mods do the same (Toybox for example).

TL;DR it takes about 30 to 60 seconds longer to load, depending on your specs. Sorry.

I can probably predefine which blueprints needs to be edited and reduce load time that way.

**Comment by Truinto** [comment]
Made huge improvement in loading time.
